src/client/Microsoft.Identity.Client/TelemetryCore/ITelemetryManager.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.Identity.Client.TelemetryCore
     internal interface ITelemetryManager
     {
         TelemetryCallback Callback { get; }
-
+        int SuccessfulSilentCallCount { get; set; }
```

Comment:
This object is not responsible for knowing telemetry details. It manages the telemetry operations. This property needs to be somewhere else.

Reply:
Probably HttpTelemetryContent

Reply:
Having looked more at this, all the telemetry data you need is the telemetry manager - in the event collections.